### DISCUSSION TITLE
Error saving workflow.: Unable to find the PackDB instance. {'ref': 'default'}

### DISCUSSION THREAD

**Discussion by ziranquliu** [StackStorm/st2#6174] [Workflows]
When I try save the workflow action，occured Error like this：
![image](https://github.com/StackStorm/st2/assets/8310637/00e3eeb8-f43d-425b-8d01-1a950c51d820)
How do I solve this problem？

**Comment by fdrab** [comment]
does the right side with the actions load for you?
